Source/WebCore/page/Quirks.cpp

```diff
@@ -1838,4 +1838,19 @@ bool Quirks::shouldIgnorePlaysInlineRequirementQuirk() const
 #endif
 }
 
+bool Quirks::canUsePartitionedCookieStorage(const URL& url) const
```

Comment:
The naming seems odd? Surely everyone can use a partitioned cookie storage?

Reply:
We currently blocks all third-party cookies, so technically no one can use partitioned cookies storage.

Reply:
I see. I hadn't thought of it this way. It makes more sense now butI still think we could come up with a clearer name.
Maybe we should indicate that this is an in-memory storage for e.g. ? Or that it only impact DOM cookies (i.e. document.cookie)

Reply:
I think a better name for the Quirk is `shouldUsePartitionedCookieStorage(...)`

Reply:
Sure, I will rename it to "canUseEphemeralPartitionedStorageForDOMCookies"